<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CETC58 TSV TEST SOFT v1.0</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionLoadData"/>
    <addaction name="actionSaveData"/>
    <addaction name="actionLoadStaticData"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>设置(&amp;S)</string>
    </property>
    <addaction name="actionADC"/>
    <addaction name="actionSIGNAL"/>
    <addaction name="actionSPAN"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>窗口(&amp;W)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_7"/>
    <addaction name="action_AboutAdcAnalyzer"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>控制</string>
    </property>
    <addaction name="actionSpiCtrl"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuSettings"/>
   <addaction name="menu_2"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidgetControlPanel">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>524287</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Control Panel</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_x">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="ControlPanel" name="controlPanel" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSaveData">
   <property name="text">
    <string>保存动态数据(S)</string>
   </property>
  </action>
  <action name="actionLoadData">
   <property name="text">
    <string>读入动态数据(R)</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出(X)</string>
   </property>
  </action>
  <action name="actionSpiCtrl">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>SPI控制(C)</string>
   </property>
  </action>
  <action name="actionFftDepth">
   <property name="text">
    <string>FFT深度(D)</string>
   </property>
  </action>
  <action name="actionSpectrum">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>频谱分析(&amp;F)</string>
   </property>
  </action>
  <action name="actionLogic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>逻辑分析(&amp;L)</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>帮助主题(&amp;T)</string>
   </property>
  </action>
  <action name="action_AboutAdcAnalyzer">
   <property name="text">
    <string>关于TSV Test Soft（&amp;A）</string>
   </property>
  </action>
  <action name="actionPowerStatus">
   <property name="text">
    <string>电源状态</string>
   </property>
  </action>
  <action name="actionADC">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>ADC类型</string>
   </property>
  </action>
  <action name="actionSIGNAL">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>时钟和信号</string>
   </property>
  </action>
  <action name="actionSPAN">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>SPAN</string>
   </property>
  </action>
  <action name="actionLoadStaticData">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>读入静态数据</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ControlPanel</class>
   <extends>QWidget</extends>
   <header>controlpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
